src/dotnet/commands/dotnet-help/HelpUsageText.cs

```diff
@@ -50,5 +50,12 @@ internal static class HelpUsageText
   --fx-version <version>            {LocalizableStrings.FxVersionDefinition}
   --roll-forward-on-no-candidate-fx {LocalizableStrings.RollForwardOnNoCandidateFxDefinition}
   --additional-deps <path>          {LocalizableStrings.AdditionalDeps}
+
+Additional tools ('dotnet <toolName> --help' for more information):
```

Comment:
Doesn't this header need to be localized?

Reply:
you are right

Reply:
Also, I'd use `dotnet [tool-name] --help` for the example, which is inline with what we give for the general usage.

Reply:
Fixed, header localized. changed to [tool-name]